app/Components/header/Header.tsx

```diff
@@ -0,0 +1,96 @@
+'use client'
+
+import React, {useEffect, useState} from 'react';
+import styles from './headerStyle.module.css';
+import { Link as ScrollLink } from 'react-scroll'
+
+const data = [
+    {
+        title: "Главное",
+        href: 'Main',
+        offsetDesk: -30,
+        offsetMob: -30
+    },
+    {
+        title: "Приглашение",
+        href: 'Invitation',
+        offsetDesk: -110,
+        offsetMob: -60
+
+    },
+    {
+        title: "Расписание",
+        href: 'Schedule',
+        offsetDesk: -100,
+        offsetMob: -65
+    },
+    {
+        title: "Контакты",
+        href: 'Contacts',
+        offsetDesk: -80,
+        offsetMob: -55
+    }
+]
```

Comment:
Maybe the same as above, to have all constants in a separate file

Reply:
done